dashboard/public/html/dashboard.html

```diff
@@ -0,0 +1,60 @@
+<md-content>
+  <md-toolbar class="md-medium-tall md-whiteframe-18dp" style="position: fixed;" layout-align="center">
+    <div class="dashboard-heading"><h2 class="md-display-1">QAT Dashboard</h2></div>
+  </md-toolbar>
+  <div class="md-medium-tall-shadow"></div>
+
+  <br>
+
+  <div layout="column" layout-align="start center">
+    <div class="dashboard-item">
+      <md-button class="md-raised md-accent" ng-click="vm.requestTestExec();">
+        New Test Execution
+      </md-button>
+    </div>
+    <md-card
+        class="dashboard-item"
+        ng-repeat="testExec in vm.testExecsMap | toArray | orderBy:'uuid':true track by testExec.uuid"
+        layout="row" layout-align="center center" layout-wrap
+        md-ink-ripple>
+      <md-card-content flex>
+        <div layout="row">
+            <span class="dashboard-item-uuid-time">
+              {{vm.getDate(testExec) | date:'medium'}}
+            </span>
+            <span class="dashboard-item-counters" flex>
+              <span ng-hide="testExec.status==='QUEUED'||testExec.status==='DRYRUN'">
+                <div>
+                  TOTAL:
+                  <span class="dashboard-item-counters-count">{{testExec.counters.total}}</span>
+                </div>
+                &nbsp;&nbsp;&nbsp;
```

Comment:
@mDibyo this seems like a hack, is there no better way to achieve this (e.g. margin / padding in CSS)